Dockerfile.development

```diff
@@ -9,7 +9,7 @@ ENV MOJ_FILE_UPLOADER_ENDPOINT   replace_this_at_build_time
 ENV TAX_TRIBUNALS_DOWNLOADER_URL replace_this_at_build_time
 ENV SENTRY_DSN                   replace_this_at_build_time
 ENV GOVUK_NOTIFY_API_KEY         replace_this_at_build_time
-ENV GA_TRACKING_ID               replace_this_at_build_time
+ENV GA_TRACKING_ID               UA-92448719-1
```

Comment:
That's kind of what I thought, thanks. Re getting it to work for me locally, I'm not even sure it did. I see no events logged in our GA dashboard from last week. Maybe it just doesn't work locally? Anyway, I shall remove the GA ID here and we can see what happens on other envs. 👍